open_schools_platform/organization_management/circles/tests/utils.py

```diff
@@ -1,23 +1,25 @@
+from django.contrib.gis.geos import Point
+
 from open_schools_platform.organization_management.circles.models import Circle
 from open_schools_platform.organization_management.circles.services import create_circle, add_student_to_circle
-from open_schools_platform.organization_management.employees.services import create_employee
-from open_schools_platform.organization_management.organizations.services import create_organization
+from open_schools_platform.organization_management.employees.tests.utils import create_test_employee
+from open_schools_platform.organization_management.organizations.tests.utils import create_test_organization
 from open_schools_platform.user_management.users.models import User
 from open_schools_platform.student_management.students.services import create_student
 
 
 def create_test_circle() -> Circle:
-    organization = create_organization(name="test_org", inn="1111111111")
-    circle = create_circle(organization=organization, name="test_circle", address="d", capacity=0, description="alalal")
+    organization = create_test_organization()
+    circle = create_circle(organization=organization, name="test_circle", address="d", capacity=0, description="alalal",
+                           location=Point(0.0, 0.0))
     return circle
```

Comment:
strange method 
firstly we should create test organization and after that create test circle
pass otganization variable to create_test_circle method -- it woulbe be clearer

Reply:
you can also define default values but give developers a chance to use other create data